playbooks/edx_sandbox.yml

```diff
@@ -37,6 +38,10 @@
       when: "'localhost' in EDXAPP_MONGO_HOSTS"
     - { role: 'edxapp', celery_worker: True }
     - edxapp
+    - role: ecommerce
+      when: COMMON_ENABLE_ECOMMERCE
+    - role: ecomworker
+      when: COMMON_ENABLE_ECOMMERCE
```

Comment:
Er, I don't think so...?

These are two separate ansible roles.  They both need to be conditionally included.

Is there a less verbose way to do this?